packages/react-native-bridge/android/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/GutenbergWebViewActivity.java

```diff
@@ -150,12 +152,26 @@ private String removeWhiteSpace(String content) {
     private void setupWebViewClient() {
         mWebView.setWebViewClient(new WebViewClient() {
             private boolean mIsRedirected;
+            private boolean mIsJetpackSsoRedirected;
 
             @Override
             public boolean shouldOverrideUrlLoading(WebView view, String url) {
-                // Set if page is redirected
-                if (!mIsRedirected) {
-                    mIsRedirected = true;
+
+                if (isJetpackSsoEnabled()) {
+                    if (!mIsJetpackSsoRedirected) {
+                        mIsJetpackSsoRedirected = true;
```

Comment:
We should really avoid referring to SSO or Jetpack directly on gutenberg repo. Is it possible to have a default implementation of this method here and override it on WPAndroid?

Reply:
Yup, you are right. Fixed.